contracts/script/IncredibleSquaringDeployer.s.sol

```diff
@@ -258,149 +238,148 @@ contract CredibleSquaringDeployer is Script, Utils {
                 abi.encodeWithSelector(
-                    blsregcoord
-                        .BLSRegistryCoordinatorWithIndices
-                        .initialize
-                        .selector,
+                    regcoord.RegistryCoordinator.initialize.selector,
                     // we set churnApprover and ejector to communityMultisig because we don't need them
-                    credibleSquaringCommunityMultisig,
-                    credibleSquaringCommunityMultisig,
-                    operatorSetParams,
-                    credibleSquaringPauserReg,
-                    // 0 initialPausedStatus means everything unpaused
-                    0
+                    incredibleSquaringCommunityMultisig,
+                    incredibleSquaringCommunityMultisig,
+                    incredibleSquaringCommunityMultisig,
+                    incredibleSquaringPauserReg,
+                    0, // 0 initialPausedStatus means everything unpaused
+                    quorumsOperatorSetParams,
+                    quorumsMinimumStake,
+                    quorumsStrategyParams
                 )
             );
         }
 
-        blsPubkeyRegistryImplementation = new BLSPubkeyRegistry(
-            registryCoordinator,
-            pubkeyCompendium
-        );
-
-        credibleSquaringProxyAdmin.upgrade(
-            TransparentUpgradeableProxy(payable(address(blsPubkeyRegistry))),
-            address(blsPubkeyRegistryImplementation)
-        );
-
-        indexRegistryImplementation = new IndexRegistry(registryCoordinator);
-
-        credibleSquaringProxyAdmin.upgrade(
-            TransparentUpgradeableProxy(payable(address(indexRegistry))),
-            address(indexRegistryImplementation)
-        );
-
-        credibleSquaringServiceManagerImplementation = new IncredibleSquaringServiceManager(
+        incredibleSquaringServiceManagerImplementation = new IncredibleSquaringServiceManager(
+            delegationManager,
             registryCoordinator,
-            slasher,
-            credibleSquaringTaskManager
+            stakeRegistry,
+            incredibleSquaringTaskManager
         );
         // Third, upgrade the proxy contracts to use the correct implementation contracts and initialize them.
-        credibleSquaringProxyAdmin.upgradeAndCall(
+        incredibleSquaringProxyAdmin.upgradeAndCall(
             TransparentUpgradeableProxy(
-                payable(address(credibleSquaringServiceManager))
+                payable(address(incredibleSquaringServiceManager))
             ),
-            address(credibleSquaringServiceManagerImplementation),
+            address(incredibleSquaringServiceManagerImplementation),
             abi.encodeWithSelector(
-                credibleSquaringServiceManager.initialize.selector,
-                credibleSquaringPauserReg,
-                credibleSquaringCommunityMultisig
+                incredibleSquaringServiceManager.initialize.selector,
```

Comment:
@samlaf it seems like this needs to be updated. I noticed that `initialize` in `ServiceManagerBase.sol` in `m2-mainnet` only accepts one argument: the service manager's owner.

Was causing some issues for me when I was trying to pass in `incredibleSquaringPauserReg` and `incredibleSquaringCommunityMultisig`